src/main/java/net/smartcosmos/edge/things/rest/template/metadata/MetadataRestConnector.java

```diff
@@ -5,39 +5,34 @@
 
 import lombok.extern.slf4j.Slf4j;
 
-import org.apache.commons.lang.BooleanUtils;
-import org.apache.commons.lang.StringUtils;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.HttpMethod;
 import org.springframework.http.RequestEntity;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
 
-import net.smartcosmos.edge.things.config.SmartCosmosEdgeThingsProperties;
 import net.smartcosmos.edge.things.domain.local.metadata.RestMetadataCreateResponseDto;
-import net.smartcosmos.edge.things.domain.local.things.RestThingCreate;
 import net.smartcosmos.edge.things.rest.RestTemplateFactory;
-import net.smartcosmos.edge.things.rest.template.SmartCosmosRequest;
+import net.smartcosmos.edge.things.rest.request.MetadataRequestFactory;
 
 @Component
 @Slf4j
+@Deprecated
```

Comment:
Deprecation of internally used class:
Please add JIRA ticket to update usages of this class to epic https://smartractechnology.atlassian.net/browse/OBJECTS-851